apps/api/src/app/notification-groups/usecases/create-notification-group/create-notification-group.usecase.ts

```diff
@@ -1,16 +1,42 @@
 import { Injectable } from '@nestjs/common';
-import { NotificationGroupRepository, NotificationGroupEntity } from '@novu/dal';
+import { NotificationGroupRepository, NotificationGroupEntity, ChangeEntityTypeEnum } from '@novu/dal';
 import { CreateNotificationGroupCommand } from './create-notification-group.command';
+import { CreateChange } from '../../../change/usecases/create-change.usecase';
+import { CreateChangeCommand } from '../../../change/usecases/create-change.command';
 
 @Injectable()
 export class CreateNotificationGroup {
-  constructor(private notificationGroupRepository: NotificationGroupRepository) {}
+  constructor(private notificationGroupRepository: NotificationGroupRepository, private createChange: CreateChange) {}
 
   async execute(command: CreateNotificationGroupCommand): Promise<NotificationGroupEntity> {
-    return await this.notificationGroupRepository.create({
+    const groups = await this.notificationGroupRepository.findOne({
+      _organizationId: command.organizationId,
+    });
```

Comment:
```suggestion
    const group = await this.notificationGroupRepository.findOne({
      _organizationId: command.organizationId,
    });
```

I think it's better to call group to avoid confusion